### PR TITLE
Update emission definitions

### PR DESCRIPTION
**Pull request by zwiebo**
Add a product based variable for the emission (Products are "Heat" and "Electricity")
Also introduced a new tag "Emission Kind" for the three Kinds of Emissions we are tracking.

### CODE REVIEW

File: definitions/variable/tag_emission_kind.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- Emission Kind:
```
Comment (danielhuppmann) [inline definitions/variable/tag_emission_kind.yaml:1]: Species is a more common name for different types of emissions or greenhouse gases...
```suggestion
- Emission Species:
```

File: definitions/variable/emissions.yaml [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
-    description: Net CO2 emissions
-    unit: Mt CO2/yr
+- Emissions|{Emission Kind}|Electricity:
+    description:  emissions emitted electricity generation {Emission Kind}
+    unit: kt
+- Emissions|{Emission Kind}|Heat:
+    description: emissions emitted heat generation {Emission Kind}
+    unit: kt
+
+- Emissions|Emission Kind:
```
Comment (danielhuppmann) [inline definitions/variable/emissions.yaml:8]:
```suggestion
- Emissions|{Emission Kind}:
```

File: definitions/variable/emissions.yaml [hunk 3]
```diff
@@ -1,23 +1,19 @@
-- Emissions|CO2:
-    description: Net CO2 emissions
-    unit: Mt CO2/yr
+- Emissions|{Emission Kind}|Electricity:
+    description:  emissions emitted electricity generation {Emission Kind}
+    unit: kt
```
Comment (danielhuppmann) [inline definitions/variable/emissions.yaml:3]:
```suggestion
    unit: {Emission Kind}
```

File: definitions/variable/tag_emission_kind.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    - CO2:
+        description: Carbon dioxide
```
Comment (danielhuppmann) [inline definitions/variable/tag_emission_kind.yaml:3]: please add the "unit" as an attribute here to keep the emission species in the unit

File: definitions/variable/emissions.yaml [hunk 5]
```diff
@@ -1,23 +1,19 @@
...
+- Emissions|{Emission Kind}|Heat:
+    description: emissions emitted heat generation {Emission Kind}
+    unit: kt
+
+- Emissions|Emission Kind:
+    description: Net {Emission Kind} emissions
+    unit: Mt/yr
+- Emissions|Emission Kind|{Sector}:
```
Comment (danielhuppmann) [inline definitions/variable/emissions.yaml:11]:
```suggestion
- Emissions|{Emission Kind}|{Sector}:
```

File: definitions/variable/emissions.yaml [hunk 6]
```diff
@@ -1,23 +1,19 @@
-- Emissions|CO2:
-    description: Net CO2 emissions
-    unit: Mt CO2/yr
+- Emissions|{Emission Kind}|Electricity:
+    description:  emissions emitted electricity generation {Emission Kind}
```
Comment (danielhuppmann) [inline definitions/variable/emissions.yaml:2]:
```suggestion
    description: Emissions of {Emission Kind} emitted from electricity generation
```

File: definitions/variable/emissions.yaml [hunk 7]
```diff
@@ -1,23 +1,19 @@
...
+    unit: kt
+- Emissions|{Emission Kind}|Heat:
+    description: emissions emitted heat generation {Emission Kind}
```
Comment (danielhuppmann) [inline definitions/variable/emissions.yaml:5]:
```suggestion
    description: Emissions of {Emission Kind} emitted from heat generation 
```

File: definitions/variable/tag_emission_kind.yaml [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+    - CH4:
+        description: Methane
+    - N2O:
+        description: Nitrous oxide
```
Comment (danielhuppmann) [inline definitions/variable/tag_emission_kind.yaml:7]: Please add an empty line at the end of this file

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Thanks, see my comments inline.

After implementing my suggestions, please also reorder the list of variables to go from generic to specific...

File: definitions/variable/tag_emission_kind.yaml (hunk 8 above)
Comment (zwiebo) [inline definitions/variable/tag_emission_kind.yaml:7]:
```suggestion
        description: Nitrous oxide
        unit: Mt N2O/yr
```

Comment (zwiebo) [inline definitions/variable/tag_emission_kind.yaml:7]:
```suggestion
        description: Nitrous oxide
        unit: Mt/yr
```

File: definitions/variable/tag_emission_kind.yaml (hunk 4 above)
Comment (zwiebo) [inline definitions/variable/tag_emission_kind.yaml:3]:
```suggestion
        description: Carbon dioxide
        unit: Mt/yr
```

File: definitions/variable/tag_emission_kind.yaml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+- Emission Kind:
+    - CO2:
+        description: Carbon dioxide
+    - CH4:
+        description: Methane
```
Comment (zwiebo) [inline definitions/variable/tag_emission_kind.yaml:5]:
```suggestion
        description: Methane
        unit: Mt/yr
```

**Review by zwiebo — COMMENTED** [review]
tried to implement your suggestions

**Review by danielhuppmann — APPROVED** [review]
Looks good, thanks!

**Pull request merged**